app/models/event_instance.rb

```diff
@@ -0,0 +1,3 @@
+class EventInstance < ApplicationRecord
+  belongs_to :event
```

Comment:
I wonder if we should put the analogous has_many in the event.

Reply:
I will add it; might as well go ahead and do it in this PR!